components/common/Glossary.js

```diff
@@ -0,0 +1,117 @@
+      understand the complexity of intimate partner violence and that harm-doers
+      or people who cause harm are often survivors themselves. We use harm-doer
+      instead of perpetrator or abuser to acknowledge their ability to heal and
+      not be indefinitely defined by their actions as well.
+    </p>
+
+    <p>
+      <b>Litigation abuse</b> occurs when a harm-doer misuses the legal system
+      to assert power and control over a survivor, including by doing things in
+      court or through their attorney to increase a survivor’s legal fees. Some
+      examples include: filing multiple groundless motions to increase costs for
+      the survivor, serving irrelevant and / or harassing discovery requests on
+      the survivor (e.g., “identify all of your sexual partners”) and asking for
+      unnecessary hearings to force the survivor to appear in court and miss
+      work / find affordable child care.
+    </p>
+
+    <p>
+      <b>“No Drop Policy”</b> refers to a policy that gives the state prosecutor
+      the sole authority to decide whether or not to prosecute a harm-doer for
+      intimate partner violence in criminal court, whether or not the survivor
+      wants to press charges or take the case to trial. Under a “no drop
+      policy”, a survivor has no say in how the state will proceed with a
+      criminal case against their harm-doer.
+    </p>
+
+    <p>
+      <b>Reasonable accommodations</b> are any changes to a job, the way a job
+      is done, or the work environment made by an employer to ensure the safety
+      of a survivor while at work. Examples of reasonable accommodations
+      include, but not limited to the ability to work from home, a modified or
+      flexible schedule, and transfer to another work facility.
+    </p>
+
+    <p>
+      <b>Special considerations for public benefits</b> exempt or defer certain
+      applicants from some program requirements and / or speed up their
+      applications. For example, under certain circumstances, a survivor’s
+      application for a public benefit may be considered more quickly or they
+      may be exempted from applicable work requirements.
+    </p>
+
+    <p>
+      <b>Survivor wealth</b> refers to a state of abundance and thriving where
+      survivors have sustaining income, savings, and credit with which to build
+      wealth and the resources to support individual, intergenerational, and
+      community healing.
+    </p>
+
+    <p>
+      <b>Torts</b> describe what happens when one person injures another. For
+      example, assault, battery, and the intentional infliction of emotional
+      distress are all torts. Tort law allows the person who was injured to sue
+      the person who caused the injury in court to recover some of the costs the
+      injury caused. Some examples of costs that can be covered in a tort case
+      include medical bills, damaged property, mental health costs, and
+      attorneys’ fees.{" "}
```

Comment:
We don't need this space, right?
```suggestion
      attorneys’ fees.
```